public_api/ddtrace/__init__.pyi

```diff
@@ -0,0 +1,8 @@
+from .monkey import patch as patch, patch_all as patch_all
+from .pin import Pin as Pin
+from .settings import _config as config
+from .span import Span as Span
+from .tracer import Tracer as Tracer
+from typing import Any
+
+tracer: Any
```

Comment:
I would think that this type would be inferred. Not sure if this is a mypy thing or a doxxie thing yet.